oaknational/Oak-Web-Application
# feat(package): latest oak compoents for d and d bug fix

## Description

Music year: 1970

- List of changes
fix refresh bug for d and d quiz questions

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-2556--oak-web-application.netlify.thenational.academy/pupils/programmes/combined-science-secondary-year-11-higher-edexcel/units/electromagnetic-waves/lessons/reflection-of-light/starter-quiz
2. Click on refresh
3. You should see the page should load without crashing

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
